circle.yml

```diff
@@ -40,7 +39,10 @@ machine:
 
 checkout:
   post:
-    - git clone --depth 1 -b ${ST2_PACKAGES_BRANCH} ${ST2_PACKAGES_REPO} /home/ubuntu/st2/st2-packages
+    - |
+      git clone --depth 1 ${ST2_PACKAGES_REPO} /home/ubuntu/st2/st2-packages
+      cd /home/ubuntu/st2/st2-packages
+      git checkout ${CIRCLE_BRANCH} || true
```

Comment:
Here we suppress `git checkout` error code to avoid race, when remote `st2-packages` branch wasn't created yet.
So it works as a fall-back to default `master` branch if there is no `vX.Y` branch in packages.